systemtest/src/test/java/io/strimzi/systemtest/performance/UserOperatorPerformance.java

```diff
@@ -242,7 +242,7 @@ public void testAliceBulkBatchUseCase(int numberOfKafkaUsersToCreate, String con
                 performanceAttributes.put(PerformanceConstants.METRICS_HISTORY, this.userOperatorMetricsGatherer.getMetricsStore()); // Map of metrics history
 
                 // Step 3: Now, it's safe to log performance data as the collection thread has been stopped
-                this.userOperatorPerformanceReporter.logPerformanceData(this.testStorage, performanceAttributes, UserOperatorPerformanceTest.REPORT_DIRECTORY + "/" + PerformanceConstants.USER_OPERATOR_ALICE_BULK_USE_CASE, ACTUAL_TIME, Environment.PERFORMANCE_DIR);
+                this.userOperatorPerformanceReporter.logPerformanceData(this.testStorage, performanceAttributes, UserOperatorPerformance.REPORT_DIRECTORY + "/" + PerformanceConstants.USER_OPERATOR_ALICE_BULK_USE_CASE, ACTUAL_TIME, Environment.PERFORMANCE_DIR);
```

Comment:
I just found it during the review of this PR (I missed it during review of the previous PRs), but why are you using the class reference for the `REPORT_DIRECTORY`? (this applies also for the TopicOperatorPerformance)

Reply:
I am not sure why I used class reference...I think it doesn't provide any additional value so I could easily remove it :)

Reply:
Yeah you removed it on two places, but there are another two :D

Reply:
🤦‍♂️